Src/Base/AMReX_VisMF.cpp

```diff
@@ -1136,20 +1135,19 @@ VisMF::Write (const FabArray<FArrayBox>&    mf,
                     Gpu::streamSynchronize();
                     fabdata = hostfab->dataPtr();
                 }
-#endif
+        #endif
                 if(doConvert) {
                     char *cDataPtr = new char[writeDataSize];
                     RealDescriptor::convertFromNativeFormat(static_cast<void *> (cDataPtr),
                                                             writeDataItems,
                                                             fabdata, *whichRD);
                     nfi.Stream().write(cDataPtr, writeDataSize);
-                    nfi.Stream().flush();
```

Comment:
There might be a problem if the flush here is removed, because cDataPtr is deleted in the next line.

Reply:
Okay, I guess this is fine, because write simply copies the data into its internal buffer.